core/src/test/java/io/questdb/griffin/SqlParserTest.java

```diff
@@ -2378,6 +2378,17 @@ public void testDuplicateTables() throws Exception {
         );
     }
 
```

Comment:
Would be good to test with "" alias . 
'' is a string literal only so it shouldn't be allowed but it's better to include it before quoted identifier behavior is fixed .